### PR TITLE
Reverted  2to3 conversion related changes 

### PR DESCRIPTION
**Pull request by mishalp-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
- [ ] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by mishalp-crest** [comment]
Hey @rgil-splunk,
As discussed with Matt in [this](https://splunk.slack.com/archives/C01F1PWD3S6/p1633949079019900) slack conversation, we are not going to convert this app to python3 as we don't have a valid asset.
We will simply decertify this app. so for that reason we have reverted python3 related changes.
Please let us know if this sounds good or we need to delete this repo and create a new one?
Also, there are [pipeline failures](https://drive.google.com/file/d/1ANSP_RYstiC4Xx2KsGlXqNlg5COU5W01/view) related to python3 changes. can you please suggest something for it?

File: berryio.json [hunk 1]
```diff
@@ -5,15 +5,14 @@
     "description": "This app supports actions for APIs on the BerryIO project for the Raspberry Pi, such as GPIO status, get and set",
     "type": "information",
     "publisher": "Splunk Community",
-    "main_module": "berryio_connector.py",
-    "app_version": "2.0.3",
+    "main_module": "berryio_connector.pyc",
```
Comment (rgil-splunk) [inline berryio.json]: this still needs to point to `.py`. builds will not contain the `.pyc` files, so install will fail otherwise

**Comment by rgil-splunk** [comment]
the python3 failure should remain without being added to the expected list so a future contributor is aware of the need to py3 convert and hopefully does so

File: exclude_files.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+*.py
```
Comment (rgil-splunk) [inline exclude_files.txt]: this excludes file should be deleted. even though this a revert, the build job itself has changed to globally exclude .pyc files, so excluding .py will result in no code files being included in the tarball

**Review by rgil-splunk — DISMISSED** [review]

**Comment by mishalp-crest** [comment]
@rgil-splunk | @ricardo-phantom | @alexa-phantom 
There is one pipeline failure due to python_version in the JSON file. so, that can be ignored.

**Review by rgil-splunk — APPROVED** [review]

**Comment by gary-phantom** [comment]
This app was created during the early Phantom days to increase the app count. 
The asset Raspberry Pi has evolved now and multiple new version sof the hardware exist. 

I would think that the best solution is simply to remove the app altogether as the current version (even it's de-certified) does not serve much purpose as most users have moved on from the original hardware version.

**Review by alexa-phantom — APPROVED** [review]

**Review by ricardo-phantom — APPROVED** [review]

**Pull request merged**